src/components/screen-header/LargeScreenHeader.tsx

```diff
@@ -0,0 +1,118 @@
+import {ThemeText} from '@atb/components/text';
+import {GlobalMessage, GlobalMessageContextType} from '@atb/global-messages';
+import {StyleSheet, useTheme} from '@atb/theme';
+import {getStaticColor, StaticColor} from '@atb/theme/colors';
+import useFocusOnLoad from '@atb/utils/use-focus-on-load';
+import React from 'react';
+import {View, ViewStyle} from 'react-native';
+import {ContentWithDisappearingHeader} from '../disappearing-header';
+import {LargeHeaderButton, LargeHeaderButtonProps} from './HeaderButton';
+
+export {AnimatedScreenHeader} from './AnimatedScreenHeader';
+
+export type LargeScreenHeaderProps = {
+  title?: string;
+  titleA11yLabel?: string;
+  /**
+   * For specifying the alert context for alerts that should be shown in this
+   * header. If no context is specified then no alerts are shown.
+   */
+  globalMessageContext?: GlobalMessageContextType;
+  style?: ViewStyle;
+  containerStyle?: ViewStyle;
+  color?: StaticColor;
+  setFocusOnLoad?: boolean;
+  headerChildren?: JSX.Element | JSX.Element[];
+  children?: JSX.Element | JSX.Element[];
```

Comment:
Just be reading these parameters names I don't understand the difference. Should probably document it.